cmd/state-handler/main.go

```diff
@@ -47,9 +47,9 @@ func main() {
 	// get name space even if not set as commandline parameter
 	namespaceName := utils.GetNamespace(*namespace)
 
-	hostName := utils.GetHostName(*hostname, kubeClient, namespaceName)
-	if hostName == "" {
-		klog.Fatalf("Failed to get host name\n")
+	hostName, err := utils.GetHostName(*hostname, kubeClient, namespaceName)
```

Comment:
Please argue with me, but I think we could use os.LookupEnv directly here in main. IMO main should be used to get whatever env variables and cli arguments and pass them to native go functions. We should try not to read any env vars inside our libraries. Would you agree?

(btw eventually we should do the same for namespaceName)

Reply:
Not going to argue, making libs depending on env is like depending on some side effects is not something that you past explicitly to the lib I will fix it, also if we move to operator-sdk maybe we have the node name in some other form.